### ISSUE TITLE
ReplicaSets are missing owner in cost allocation 

### ISSUE THREAD

**Issue by dwbrown2** [kubecost/cost-analyzer-helm-chart#1048]
ReplicaSet cost metrics correctly display in Cost Allocation but they are being shown as `__unallocated__`. 

Sneaking in a feature request, it would be really nice to be able to drill in `__unallocated__`

![image](https://user-images.githubusercontent.com/298359/131376588-4b0f00e9-c992-47b1-be4e-0c770613e049.png)

gz#743

**Comment by dwbrown2** [comment]
Sample RS if you want to repro again Niko. 

```
apiVersion: apps/v1
kind: ReplicaSet
metadata:
  name: frontend
  labels:
    app: guestbook
    tier: frontend
spec:
  # modify replicas according to your case
  replicas: 3
  selector:
    matchLabels:
      tier: frontend
  template:
    metadata:
      labels:
        tier: frontend
    spec:
      containers:
      - name: php-redis
        image: gcr.io/google_samples/gb-frontend:v3
```

**Comment by kbrwn** [comment]
This was fixed in v1.87.0 closing.
